pkg/component/vpa/vpa.go

```diff
@@ -244,6 +244,9 @@ func (v *vpa) emptyMutatingWebhookConfiguration() *admissionregistrationv1.Mutat
 		suffix = "target"
 	}
 
+	// This name intentionally starts with a 'v', such that the webhook is called after all other webhooks which inject
+	// containers. All containers injected by webhooks that are called _after_ the vpa webhook will not be under control
+	// of vpa.
```

Comment:
I already assumed that the name intentionally started with `v` because `vpa` starts with `v`, but jokes aside is this assuming/knowing that webhooks are executed in alphabetical order? If so, why not state this instead? Why not go further and prefix the name with `zzz-`?

Reply:
done.